WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -746,7 +769,7 @@ def __startJobAgent(self):
 
     retCode, _output = self.executeAndGetOutput( jobAgent, self.pp.installEnv )
     if retCode:
-      self.log.error( "Could not start the JobAgent" )
+      self.log.error( "Error executing the JobAgent [ERROR %d]" % retCode )
```

Comment:
I can do that as generic, i.e. every time there's the need to call to sys.exit(1). What do you think?